src/components/post-form.js

```diff
@@ -5,7 +5,7 @@ import styled from 'styled-components';
 import {LoadingOutlined} from '@ant-design/icons';
 import slogan from '../assets/slogan.svg';
 import uploadIcon from '../assets/upload-icon.svg';
-
+import {fileToBase64ByQuality, convertBase64UrlToBlob} from '../utils/image-compression';
```

Comment:
An empty line required.

```suggestion
import {fileToBase64ByQuality, convertBase64UrlToBlob} from '../utils/image-compression';

```